.github/workflows/ci.yml

```diff
@@ -12,43 +12,26 @@ jobs:
       fail-fast: false
       matrix:
         versions:
-          - python: 3.5
-            toxenv: py35-2.2.X
-          - python: 3.6
-            toxenv: py36-2.2.X
-          - python: 3.7
-            toxenv: py37-2.2.X
           - python: 3.8
-            toxenv: py38-2.2.X
-          - python: 3.9
-            toxenv: py39-2.2.X
-
-          - python: 3.6
-            toxenv: py36-3.0.X
-          - python: 3.7
-            toxenv: py37-3.0.X
+            toxenv: py38-3.2.X
           - python: 3.8
-            toxenv: py38-3.0.X
-          - python: 3.9
-            toxenv: py39-3.0.X
-
-          - python: 3.6
-            toxenv: py36-3.1.X
-          - python: 3.7
-            toxenv: py37-3.1.X
+            toxenv: py38-4.0.X
           - python: 3.8
-            toxenv: py38-3.1.X
-          - python: 3.9
-            toxenv: py39-3.1.X
-
-          - python: 3.6
-            toxenv: py36-3.2.X
-          - python: 3.7
-            toxenv: py37-3.2.X
-          - python: 3.8
-            toxenv: py38-3.2.X
+            toxenv: py38-4.2.X
+         
           - python: 3.9
             toxenv: py39-3.2.X
+          - python: 3.9
+            toxenv: py39-4.0.X
+          - python: 3.9
+            toxenv: py39-4.2.X
+        
+          - python: 3.10
+            toxenv: py310-3.2.X
+          - python: 3.10
+            toxenv: py310-4.0.X
+          - python: 3.10
+            toxenv: py310-4.2.X
```

Comment:
You need to add quotes around `3.10`, otherwise yaml parses it as `3.1`.

Reply:
Done!